<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParameterSetting</class>
 <widget class="QDialog" name="ParameterSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>260</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>AR PL UKai CN</family>
    <pointsize>17</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>参数设置</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../src/image.qrc">
    <normaloff>:/images/leiwo.png</normaloff>:/images/leiwo.png</iconset>
  </property>
  <widget class="SoftKeyLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>39</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/setting/cut_height_min.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_103">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>39</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/con_recong/ProfilingHeightSet.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>39</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/setting/cut_height_max.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_102">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>39</width>
     <height>39</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/setting/Weight1000.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>39</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/setting/RollerSpeedLowSetting.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="SoftKeyLineEdit" name="l_concaveclset">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>39</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/setting/ConcaveClearanceSetting.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="SoftKeyLineEdit" name="l_rsls">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="SoftKeyLineEdit" name="l_cuth_max">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="SoftKeyLineEdit" name="l_pfh">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="SoftKeyLineEdit" name="l_cuth_min">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>70</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>rpm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>mm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>g</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>mm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>mm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>mm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OK_btn">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>200</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(58, 111, 249);</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel_btn">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>200</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(249, 88, 88);</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="grain_kind">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>100</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>水稻</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>小麦</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>玉米</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>大豆</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>其他</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>20</y>
     <width>39</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/setting/CutMachineMode.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_105">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>110</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/engine/RiceKind.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="cbx_2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>20</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>切碎</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>排草</string>
    </property>
   </item>
  </widget>
  <widget class="SoftKeyLineEdit" name="rollerSpeed">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="SoftKeyLineEdit" name="draughtFanSpeed">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="SoftKeyLineEdit" name="sortingScreenUp">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="SoftKeyLineEdit" name="sortingScreenDown">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>80</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="SoftKeyLineEdit" name="sortingScreenTail">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>140</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/images/engine/RollerSpeed.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/images/engine/draught_fanimpurity_rate.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/parameters/sorting_screen_up.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>90</y>
     <width>40</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/parameters/sorting_screen_down.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>140</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/parameters/sorting_screen_tail.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>140</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>rpm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>rpm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>80</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>mm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>140</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>mm</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SoftKeyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>softkeylineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../src/image.qrc"/>
 </resources>
 <connections/>
</ui>
